app/main.py

```diff
@@ -1 +1,16 @@
-# write your code here
+import os
+import shutil
+
+
+def move_file(command: str) -> None:
+    command = command.split()
+    if len(command) != 3 or command[0] != "mv":
+        raise ValueError(
+            "Vanga (not tests or task) have told me I have to make this logic"
```

Comment:
Not Vanga, but checklist requires to verify that the command is correct ))
![image](https://github.com/mate-academy/py-move-file/assets/44442087/ee5c45bd-bd5f-40dd-a924-a2c01555ca15)